BlogApp/src/BlogItem.js

```diff
@@ -1,22 +1,29 @@
-import React from 'react';
-import { getUser } from './service';
+import React from "react";
+import { getUser } from "./service";
+import styles from "./styles";
 
 export default function BlogItem(props) {
   const { blog = {} } = props || {};
   const user = getUser(blog.userId) || {};
-  const { fullName = 'Muhammad' } = user;
+  const { fullName = "Muhammad" } = user;
   return (
-    <div>
-      <div>
-        <div>{fullName[0]}</div>
-        <div>User: {user.fullName}</div>
-        <div>Timestamp: {blog.timestamp}</div>
+    <div style={styles.blogItemContainer}>
+      <div style={styles.blogItemHeader}>
+        <div style={styles.profilePicture}> {fullName[0]} </div>
+        <div style={styles.profileName}>
+          <b> {user.fullName} </b>
+        </div>
       </div>
-      <div>Text: {blog.text}</div>
-      <div>
-        <button>Edit</button>
-        <button>Delete</button>
+      <div style={styles.blogItemMainText}> {blog.text} </div>
+      <div style={styles.blogItemFooter}>
+        <div style={styles.blogItemTimestamp}>
+          Posted on: March 20, 2019{/*blog.timestamp*/}
```

Comment:
I think the following should be replaced later on to get the real date of each blog post !! 😇

```suggestion
          Posted on: {blog.timestamp}
```